nautobot/core/templates/redoc_ui.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block title %}{{ title | default:"Redoc" }}{% endblock title %}
+
+{% block breadcrumbs %}
+    {% include "inc/breadcrumbs.html" %}
+{% endblock %}
+
+{% block extra_styles %}
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;700&family=Roboto:wght@300;400;700&display=swap">
```

Comment:
Nautobot should be installable/usable in airgapped environments - can we statically package these fonts easily? Clearly this is probably not a *new* problem, but we should fix it if it's not too much trouble.

Reply:
Let's just serve the fonts from static for now. Thanks for investigating.